web/sharings/sharings.go

```diff
@@ -733,7 +733,10 @@ func checkRevokePermissions(c echo.Context, ins *instance.Instance, sharing *sha
 		return ins.PickKey(token.Claims.(*permissions.Claims).Audience)
 	}, &claims)
 	if err != nil {
-		return err
+		return sharings.ErrForbidden
```

Comment:
Out of curiosity why do you return `ErrForbidden` here? Because it would give away information?